cg/services/post_processing/error_handlers.py

```diff
@@ -0,0 +1,27 @@
+"""Module that holds the error handlers for the Post Processing service."""
+
+import logging
+from cg.exc import CgError
+
+LOG = logging.getLogger(__name__)
+
+
+def handle_post_processing_errors(to_except: tuple, to_raise):
+    """Handle errors for the Post processing services."""
+
+    def error_handling(fn):
```

Comment:
Nice! can't this be `@wraps(func)`?

Reply:
It complains of not getting a parameter. I will read up on the proper usage.